scripts/src/main/resources/scripts/command/docker

```diff
@@ -16,46 +16,125 @@ source "$(dirname "${0}")"/../functions
 
 # $1: optional setup
 function doSetup() {  
-  if doIsDockerInstalled 
+  if [ "${DOCKER_EDITION}" != "DOCKERDESKTOP" ]
   then
-    if [ "${1}" != "silent" ] && ! doIsQuiet
-    then
-      doEcho "docker is already installed at $(command -v docker)"
-      docker version
+    echo "Rancher Desktop set as option"
+    if uname -r | grep -i microsoft
+    then 
+      echo "It seems that you are running the setup from inside a WSL shell."
+      echo "Please use git-bash for that purpose. Otherwise your OS will be interpreted as Linux"
+    else
+      if doIsRancherInstalled 
+      then
+        doIsWindows 
+        queryWin=$?
+        if [ "${1}" != "silent" ] && ! doIsQuiet
+        then
+          #give the User the installation location when using linux or mac
+          if [ $queryWin != 0 ]
+          then
+            doEcho "Rancher Desktop is already installed at $(command -v devon docker)"
+          else
+            doEcho "Rancher Desktop is already installed"
+          fi
+        fi
+      else
+        doRequireNotBatch
+        local latestversion
+        latestversion=$(curl -Ls -o /dev/null -w '%{url_effective}' https://github.com/rancher-sandbox/rancher-desktop/releases/latest | sed 's/.*\([0-9]\.[0-9]\.[0-9]\).*/\1/')
+        local previousversion="1.2.1"
+        if [[ "${latestversion}" =~ ^[0-9]\.[0-9].[0-9]$ ]] 
+        then
+          if [ "${latestversion}" \> "${previousversion}" ]
```

Comment:
is this comparison syntax working reliably?
We have implemented our own function `doVersionCompare` in `functions` that was quite complex.
If this works as well, we could strip out `doVersionCompare` and replace it with this simplified syntax.
Otherwise, if this does not work for version numbers containing letters, `alpha`, `beta`, or multiple dots we should also use `doVersionCompare` here for consistency and correctness.

Reply:
Looks as if `doVersionCompare` has more power that my code-snippet for this one special case does (also there was a typo, another backslash would be needed before the second dot, to only accept dots in between the numbers). 
Therefore I switched to using this function and keep its usage in mind for future use cases. 
Thanks for the detailed feedback.

Reply:
sorry, I must have missed to publish the changes after commiting them last week. Now the changes are also published.

Reply:
Now, the commit arrived and I can resolve this.